web/src/routes/__tests__/SearchPage.spec.tsx

```diff
@@ -1,17 +1,23 @@
 import { fireEvent } from '@testing-library/react'
-import { CategoriesMapModel, CategoryModel, SEARCH_ROUTE } from 'api-client'
-import CityModelBuilder from 'api-client/src/testing/CityModelBuilder'
+import {
+  CategoriesMapModel,
+  CategoriesMapModelBuilder,
+  CategoryModel,
+  CityModelBuilder,
+  LanguageModelBuilder,
+  SEARCH_ROUTE
+} from 'api-client'
 import moment from 'moment'
 import React from 'react'
 import { Route } from 'react-router-dom'
 import { ThemeProvider } from 'styled-components'
-import LanguageModelBuilder from '../../../../api-client/src/testing/LanguageModelBuilder'
 import buildConfig from '../../constants/buildConfig'
 import { mockUseLoadFromEndpointWitData } from '../../testing/mockUseLoadFromEndpoint'
```

Comment:
Didn't someone move this to the api-client?

Reply:
Not yet up to date with main 🙈